### PR TITLE
OBPIH-1559 Show quantity received only if confirmed receipt

### PR DESCRIPTION
**Pull request by DominikaHatala**

### CODE REVIEW

File: grails-app/domain/org/pih/warehouse/receiving/ReceiptItem.groovy [hunk 1]
```diff
@@ -63,9 +63,13 @@ class ReceiptItem implements Comparable<ReceiptItem>, Serializable {
 		isSplitItem(nullable:true)
 		comment(nullable:true, maxSize: 255)
 	}
-	
+
+	def getReceivedQuantity() {
+		return receipt.receiptStatusCode == ReceiptStatusCode.RECEIVED ? quantityReceived : 0
```
Comment (jmiranda) [inline grails-app/domain/org/pih/warehouse/receiving/ReceiptItem.groovy]: Having two getter methods to get received quantity seems like a recipe for disaster. Let's not do this.

Comment (jmiranda) [reply grails-app/domain/org/pih/warehouse/receiving/ReceiptItem.groovy]: So remove this method and ...

File: grails-app/views/stockMovement/_receipts.gsp [hunk 2]
```diff
@@ -48,7 +48,7 @@
                     ${receiptItem?.binLocation?.name}
                 </td>
                 <td>
-                    ${receiptItem?.quantityReceived?:0}
+                    ${receiptItem?.getReceivedQuantity()?:0}
```
Comment (jmiranda) [inline grails-app/views/stockMovement/_receipts.gsp]: ... instead, add the conditional around the view.
```
<g:if test="${receipt.receiptStatusCode == ReceiptStatusCode.RECEIVED}">
${receiptItem?.quantityReceived?:0}
</g:if>
<g:else>
<g:message code="default.tbd.label" default="TBD"/>
</g:else>
```

Also, in general, there's no need for the get in getReceivedQuantity. The following are equivalent.
```
${receiptItem?.getReceivedQuantity()?:0}
```
and 
${receiptItem?.receivedQuantity?:0}

Comment (jmiranda) [reply grails-app/views/stockMovement/_receipts.gsp]: For the "TBD" part, in the ticket ask Jesse what she wants there. Maybe show a screenshot of what it looks like and ask if she'd prefer a blank cell, 0, or something like TBD, which is short for "To be determined".

Comment (DominikaHatala) [reply grails-app/views/stockMovement/_receipts.gsp]: @jmiranda I've added changed commit, could you review?

**Pull request merged**